src/epoll.rs

```diff
@@ -105,7 +111,7 @@ impl Poller {
     }
 
     /// Modifies an existing file descriptor.
-    pub fn modify(&self, fd: RawFd, ev: Event, mode: PollMode) -> io::Result<()> {
+    pub fn modify(&self, fd: BorrowedFd<'_>, ev: Event, mode: PollMode) -> io::Result<()> {
```

Comment:
If `add` is `unsafe` and takes a `RawFd`, then `modifiy` should be too, for the same reason.

Reply:
Ah, good point. I was somehow thinking it could modify the stored file descriptor, but as you say, it can't.